src/spatialdata/_io/io_raster.py

```diff
@@ -80,7 +81,7 @@ def _read_multiscale(
                 data,
                 name=name,
                 dims=axes,
-                coords={"c": channels} if raster_type == "image" else {},
+                coords={"c": channels} if channels is not None else {},
```

Comment:
thanks! when is this prolbem arising?

Reply:
I think I triggered this by trying to open files saved with the old omero metadata.